<asm/types.h> doesn't define __u64 in __STRICT_ANSI__ (-ansi) mode.  In
order to use <linux/cdrom.h> in this mode we need to fix this file not to
assume __u64.

--- include/asm-i386/byteorder.h.orig	2003-10-15 11:13:06.000000000 -0400
+++ include/asm-i386/byteorder.h	2003-11-01 17:33:12.000000000 -0500
@@ -34,6 +34,10 @@
 		return x;
 }
 
+#define __arch__swab32(x) ___arch__swab32(x)
+#define __arch__swab16(x) ___arch__swab16(x)
+
+#if !defined (__STRICT_ANSI__)
 
 static inline __u64 ___arch__swab64(__u64 val) 
 { 
@@ -55,10 +59,9 @@
 } 
 
 #define __arch__swab64(x) ___arch__swab64(x)
-#define __arch__swab32(x) ___arch__swab32(x)
-#define __arch__swab16(x) ___arch__swab16(x)
 
 #define __BYTEORDER_HAS_U64__
+#endif
 
 #endif /* __GNUC__ */
 
